contracts/ens/PodEnsRegistrar.sol

```diff
@@ -147,13 +148,30 @@ contract PodEnsRegistrar is Ownable {
         ens.setSubnodeRecord(rootNode, label, owner, address(resolver), 0);
     }
 
+    function deregister(address safe, bytes32 label)
+        external
+        onlyControllerOrOwner
+    {
+        // Ensure that the label provided matches the safe we are deregistering.
+        // Otherwise, you can deregister other labels by deregistering an arbitrary pod.
+        bytes32 node = getEnsNode(label);
+        address addr = resolver.addr(node);
+        require(addr == safe, "safe and label didn't match");
+        reverseRegistrar.setName("");
```

Comment:
i think that this is clearing out the reverseRegistrar of PodEnsRegistrar.sol. The function in the `safeTeller` should handle zeroing out the reverse registrar of the safe

Reply:
yeah you're right, good catch!